dev/faq/tidb-lightning.md

```diff
@@ -28,12 +28,14 @@ TiDB Lightning requires the following privileges:
 * CREATE
 * DROP
 
-If the target database is used to store checkpoints, it additionally requires these privileges:
+If the [TiDB-back-end](/dev/reference/tools/tidb-lightning/tidb-backend.md) is chosen, or the target database is used to store checkpoints, it additionally requires these privileges:
```

Comment:
TiDB-backend

Reply:
No need to use double hyphens. Looks strange. The same with import-backend

Reply:
@dcalvin But "back end" is spelled with a space. Replace all "back end" to "backend" too?

Reply:
ping @dcalvin ,PTAL

Reply:
Sound good to me. @kennytm Sorry for delayed response.